src/store/getters.js

```diff
@@ -117,4 +117,7 @@ export default {
   tourRunning({ tourRunning }) {
     return tourRunning;
   },
+  tourStartBar({ tourStartBar }) {
+    return tourStartBar;
+  },
```

Comment:
I know this kind of agreement in this repo, but getters that just wrapping a single state variable have no sense

Reply:
Shoud I use directly $store.getters

Reply:
I propose to use `$store.state.tourStartBar` instead of wrapping it (`$store.getters. tourStartBar`)
e.g. map it to components using `mapState`

Reply:
Yeah my mistake I wanted to say `$store.state` not `$storage.getters`